frontend/packages/wizard/src/wizard.tsx

```diff
@@ -135,11 +137,17 @@ const Wizard = ({
   };
 
   const handleNext = () => {
-    dispatch(WizardAction.NEXT);
+    dispatch({ type: WizardActionType.NEXT });
+  };
+
+  const handleStepClick = (step: number) => {
+    dispatch({ type: WizardActionType.GO_TO_STEP, step });
   };
 
   const context = (child: JSX.Element) => {
     return {
+      getNextStepToComplete: () => state.nextStepToComplete,
```

Comment:
Im not seeing the updates to the WizardContext which this is based off of, so we'll need to update that to support the additional functionality.